docker-compose.yml

```diff
@@ -869,6 +870,25 @@ services:
     volumes: *ubuntu-volumes
     command: *python-command
 
+  ubuntu-swift:
+    # Usage:
+    #   docker-compose build ubuntu-swift
+    #   docker-compose run --rm ubuntu-swift
+    # Parameters:
+    #   ARCH: amd64, arm64v8, ...
+    #   UBUNTU: 20.04
+    image: ${REPO}:${ARCH}-ubuntu-${UBUNTU}-swift
+    build:
+      context: .
+      dockerfile: ci/docker/ubuntu-${UBUNTU}-swift.dockerfile
+    shm_size: *shm-size
+    environment:
+      <<: *ccache
+    volumes: *ubuntu-volumes
+    command: >
+      /bin/bash -c "/arrow/ci/scripts/swift_test.sh /arrow"
```

Comment:
I have no experience with swift but don't we have to build first? I can't see us using the added `swift_build.sh` script anywhere, is it necessary?

Reply:
Good point, "swift test" will start the build.  I will remove the swift_build.sh script.